src/aks-preview/setup.py

```diff
@@ -9,7 +9,7 @@
 
 from setuptools import setup, find_packages
 
-VERSION = "0.5.166"
+VERSION = "0.5.167"
```

Comment:
The extension version  `0.5.167` is already exists

Reply:
resolved, thanks